paths.json

```diff
@@ -1,6 +1,9 @@
 {
   "mappings": [
-    "/content/pricefx/en:/"
+    "/content/pricefx/:/",
+    "/content/pricefx/en:/",
+    "/content/exlm/en/configuration/redirects.json",
+    "/content/pricefx.resource/en/configuration/query-index.json:/en/configuration/query-index.json"
```

Comment:
Can you change this to "/content/pricefx.resource/en/configuration/query-index.json:/configuration/query-index.json"

Reply:
done